### DISCUSSION TITLE
Privacy mode feature support (Amcrest IP4M-1041)

### DISCUSSION THREAD

**Discussion by csw3190** [blakeblackshear/frigate#5886] [Ideas]
I have two IP4M-1041 cameras in my house pointed at front and rear entryways. These cameras have a "privacy mode," which points the camera down (obscuring the lens) and seems to disconnect the RTSP stream. 

As a result, when privacy mode is enabled, Frigate just errors out and can't connect.

I have privacy mode exposed as a switch in HA via the Amcrest integration (YAML). It automatically turns ON when we arrive or are home. Privacy mode turns OFF when we leave so the cameras are active.

Is there a way to tell Frigate to only engage the RTSP stream in certain cases (i.e., when privacy mode is off)? If not, I think this would be a useful feature to identify if a human is detected indoors while people are away.

**Comment by NickM-27** [comment]
There is a pinned feature request https://github.com/blakeblackshear/frigate/issues/1911
